lib/utils/constants.dart

```diff
@@ -1,11 +1,11 @@
 import 'package:flutter/material.dart';
 
 class CometChatConstants {
-  static const String appId = "208434241880dc4d"; //"210953e5858805fa";
-  static const String authKey =
-      "a391c56883a69ab7421485b35a086628009f541a"; //"f063921eb9b7d7076ec3aa728c73dc15fe241ac6";
-  static const String region = "us";
-}
+   static const String appId = "2379614bd4db65dd";
```

Comment:
These need to be placeholders and not actual text

Reply:
@priyadarshininadar I have made the requested changes